### ISSUE TITLE
[Bug] Cannot edit attributes

### ISSUE THREAD

**Issue by LeKopic** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#323]
When I try to edit any attributes (weapons, feats, talents, etc.) - the system just doesn't save the changes. There are no errors in the console.
![image](https://user-images.githubusercontent.com/42682507/194744204-9a3c0816-3149-4403-a550-90173328ac23.png)

**Comment by LeKopic** [comment]
Hm, looking at it closer - I am not sure it is an issue with the system and not the foundry itself. I will do some testing today.

**Comment by LeKopic** [comment]
Yep, the problem exists even in a new game world, so it is not connected to any enabled modules.

**Comment by LeKopic** [comment]
And two more things that might be connected to this issue:
1) When I try to add class bonus feats the system always considers it to be a general feat.
2) When I sometimes add two GM bonuses (and maybe other traits) to a character it only adds the number of bonuses I used for the first GM Bonus. Example:
I added GM Bonus (General Feats, 10) first, then I added GM Bonus (General Feats, 152) and it still counted it as 10
![image](https://user-images.githubusercontent.com/42682507/194765801-7be170f9-a1f7-4364-9954-85bd65564722.png)
In the pic - GM Bonus (General Feats, 152) is shown.
![image](https://user-images.githubusercontent.com/42682507/194765862-74dec2c4-3305-4998-ad37-dd7ae6b166f5.png)
